src/js/stores/ChronosStore.js

```diff
@@ -66,10 +66,7 @@ class ChronosStore extends EventEmitter {
       return true;
     });
 
-    VisibilityStore.addChangeListener(
-      VISIBILITY_CHANGE,
-      this.onVisibilityStoreChange.bind(this)
-    );
+    this.store_initializeListeners([{name: 'visibility', events: ['change']}]);
```

Comment:
Do we really wan't to introduce the store mixin only to add a listener to the visibility store? I thought our overall plan is to "remove" mixins...  I'd rather introduce something like the recurrent task util (https://github.com/dcos/dcos-ui/pull/91) to handle this in a clean consistent way. What do you think?